src/components/Api/api.tsx

```diff
@@ -0,0 +1,25 @@
+const getDataCard = async (text: string) => {
+  try {
+    const resp = await fetch(`https://rickandmortyapi.com/api/character/?name=${text}`);
```

Comment:
create a baseUrl constant where u will store 'https://rickandmortyapi.com/api/character' because your requests use it